src/ui/public/chrome/_variables.scss

```diff
@@ -1,5 +1,5 @@
 
-$kbnChromeGlobalNavBackground: shade($euiColorPrimary, 30%);
+$kbnChromeGlobalNavBackground: #005571;
```

Comment:
Remember that hardcoding hexcodes is a send-to-hell-able offense.

Reply:
In the K7 world! This code dissapears in K7 remember

;-)